packages/rainbow-sprinkles/src/__tests__/createRainbowSprinkles.test.ts

```diff
@@ -30,373 +30,273 @@ const vars = {
-    expect(result.color).toMatchObject([
-      {
-        classes: {
-          dynamic: {
-            mobile: 'color-mobile',
-            desktop: 'color-desktop',
-            tablet: 'color-tablet',
-          },
-        },
-        name: 'color',
-        scale: {
-          gray100: '#fefefe',
-          gray200: '#333333',
-          gray50: '#efefef',
-          white: '#fff',
-        },
-        vars: {
-          desktop: '--color-desktop',
-          mobile: '--color-mobile',
-          tablet: '--color-tablet',
-        },
-      },
-    ]);
-    expect(result.paddingX).toMatchObject([
-      {
-        classes: {
-          dynamic: {
-            mobile: 'paddingLeft-mobile',
-            tablet: 'paddingLeft-tablet',
-            desktop: 'paddingLeft-desktop',
-          },
-        },
-        name: 'paddingLeft',
-        scale: {
-          '1x': '5px',
-          '2x': '10px',
-          '3x': '15px',
-          none: '0',
-        },
-        vars: {
-          desktop: '--paddingLeft-desktop',
-          mobile: '--paddingLeft-mobile',
-          tablet: '--paddingLeft-tablet',
-        },
-      },
-      {
-        classes: {
-          dynamic: {
-            desktop: 'paddingRight-desktop',
-            mobile: 'paddingRight-mobile',
-            tablet: 'paddingRight-tablet',
-          },
-        },
-        name: 'paddingRight',
-        scale: {
-          '1x': '5px',
-          '2x': '10px',
-          '3x': '15px',
-          none: '0',
-        },
-        vars: {
-          desktop: '--paddingRight-desktop',
-          mobile: '--paddingRight-mobile',
-          tablet: '--paddingRight-tablet',
-        },
-      },
-    ]);
+  const rainbowSprinkles = createRainbowSprinkles({
+    conditions: {
+      mobile: {},
+      tablet: { '@media': 'screen and (min-width: 768px)' },
+      desktop: { '@media': 'screen and (min-width: 1024px)' },
+    },
+    defaultCondition: 'mobile',
+    dynamicProperties: {
+      display: true,
+      flexDirection: true,
+      alignItems: true,
+      justifyContent: true,
+      gap: vars.space,
+      padding: vars.space,
+      paddingLeft: vars.space,
+      paddingRight: vars.space,
+      paddingTop: vars.space,
+      paddingBottom: vars.space,
+      width: true,
+      height: true,
+      borderRadius: vars.borderRadius,
+      fontFamily: vars.fontFamily,
+      fontSize: vars.fontSize,
+      textAlign: true,
+      color: vars.color,
+      background: vars.color,
+    },
+    shorthands: {
+      p: ['padding'],
+      pl: ['paddingLeft'],
+      pr: ['paddingRight'],
+      pt: ['paddingTop'],
+      pb: ['paddingBottom'],
+      paddingX: ['paddingLeft', 'paddingRight'],
+      paddingY: ['paddingTop', 'paddingBottom'],
+      px: ['paddingLeft', 'paddingRight'],
+      py: ['paddingTop', 'paddingBottom'],
+      bg: ['background'],
+      placeItems: ['alignItems', 'justifyContent'],
+    },
   });
 
   test('getBoxProps', () => {
```

Comment:
Thoughts on renaming this and other describe blocks from `getBoxProps` to `rainbowSprinkles`?